packages/story-editor/src/components/videoTrim/videoTrimmer.js

```diff
@@ -165,7 +165,11 @@ function VideoTrimmer() {
       <RailWrapper>
         <Rail
           width={railWidth}
-          style={{ backgroundImage: `url(${railBackgroundImage})` }}
+          style={{
+            backgroundImage: railBackgroundImage
+              ? `url(${railBackgroundImage})`
+              : undefined,
```

Comment:
Shouldn't it be `none`? 🤔 Or if it's `undefined`, maybe it's not displayed at all in the DOM?

Reply:
It won't be displayed at all.

Here's the DOM when it's missing:
![Screenshot 2022-01-19 at 11 37 11](https://user-images.githubusercontent.com/841956/150114067-61003b79-2b02-4489-be97-a0c60e6efc5c.png)

Here's when it arrives:
![Screenshot 2022-01-19 at 11 37 47](https://user-images.githubusercontent.com/841956/150114053-d128d847-ca16-4954-8a58-001d61b77fc4.png)